Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -2068,6 +2082,7 @@ def run(self):
             platform = platform.split('-')[0]
         if platform in ('mac', 'ios', 'visionos', 'tvos', 'watchos', '*'):
             command_list.extend(self.command_list_apple)
+            command_list[5:5] = [self.shell_command('echo TimezoneVers: $(cat /usr/share/zoneinfo/+VERSION)')]
```

Comment:
nit: I wonder if `echo TimezoneVers:` is really necessary. I don't see anywhere in the codebase parsing for it, and stdout logs show the command as it's being run, so I don't think printing "TimezoneVers:" adds meaningful clarity. 

cc @robert-jenner @JonWBedard as authors of https://commits.webkit.org/259847@main. Perhaps we can remove the `echo` and then treat this like a normal part of the command list?

Reply:
Given we do indeed seem not to use this anywhere, I've just gone ahead and removed the `echo`.